### PR TITLE
Showing error messages

### LINKED ISSUE #11: Add basic error handling in the UI
Currently errors are only printed in the debug console. We should show a basic error dialog/toast with an error message, so that we can see errors even without access to the debug console (for example when running a Testflight build).

### PR DESCRIPTION
**Pull request by arturjaworski**
Resolves #11

### CODE REVIEW

File: Customer Demo/Customer Demo/Screens/Setup/SetupViewController.swift [hunk 1]
```diff
@@ -59,6 +62,13 @@ class SetupViewController: UIViewController {
 
         navigationController?.pushViewController(subscriberStatusViewController, animated: true)
     }
+    
+    func showValidationError() {
+        let alertController = UIAlertController(title: "Validation error", message: "Please fill out all fields", preferredStyle: .alert)
+        alertController.addAction(UIAlertAction(title: "OK", style: .default))
+        
+        self.present(alertController, animated: true)
```
Comment (JakubJankowski) [inline Customer Demo/Customer Demo/Screens/Setup/SetupViewController.swift]: I don't think you need to add `self` here ;) Same in the `showError` method below.

**Review by JakubJankowski — APPROVED** [review]

File: Customer Demo/Customer Demo/Screens/SubscriberStatus/SubscriberStatusViewController.swift [hunk 2]
```diff
@@ -185,6 +185,13 @@ class SubscriberStatusViewController: UIViewController {
         
         mapTypeButton.setImage(image, for: .normal)
     }
+    
+    func showError(_ error: ErrorInformation) {
+        let alertController = UIAlertController(title: "Error", message: error.message, preferredStyle: .alert)
```
Comment (JakubJankowski) [inline Customer Demo/Customer Demo/Screens/SubscriberStatus/SubscriberStatusViewController.swift:190]: I don't think that the ViewController needs the whole ErrorInformation object here. It uses only the `error.message` string here, so might as well just pass the String itself as a parameter. It's a minor thing though, feel free to ignore if you don't feel strongly about this.

File: Customer Demo/Customer Demo/Screens/Setup/SetupViewController.swift (hunk 1 above)
Comment (arturjaworski) [reply Customer Demo/Customer Demo/Screens/Setup/SetupViewController.swift]: Removed

File: Customer Demo/Customer Demo/Screens/SubscriberStatus/SubscriberStatusViewController.swift (hunk 2 above)
Comment (arturjaworski) [reply Customer Demo/Customer Demo/Screens/SubscriberStatus/SubscriberStatusViewController.swift:190]: It was intended. `ErrorInformation` is just a generic struct. I don't think it shouldn't be passed to view.

**Pull request merged**